src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -1474,7 +1477,9 @@ def db_import(
     kwargs['storage_key_type'] = storage_key_type
     kwargs['storage_key'] = storage_key
 
-    return client.begin_import_method(
+    return sdk_no_wait(
+        no_wait,
```

Comment:
Please add `no_wait` parameter for func `db_import()`

The `no_wait` used here is not defined